cupy/statistics/correlation.py

```diff
@@ -48,7 +49,32 @@ def corrcoef(a, y=None, rowvar=True, bias=None, ddof=None):
     return out
 
 
-# TODO(okuta): Implement correlate
+def correlate(a, v, mode='valid'):
+    """Returns the cross-correlation of two 1-dimensional sequences.
+
+    Args:
+        a (cupy.ndarray): first 1-dimensional input.
+        v (cupy.ndarray): second 1-dimensional input.
+        mode (str, optional): `valid`, `same`, `full`
+
+    Returns:
+        cupy.ndarray: Discrete cross-correlation of a and v.
+
+    .. seealso:: :func:`numpy.correlate`
+
+    """
+    if a.ndim != 1 or v.ndim != 1:
+        raise ValueError('object too deep for desired array')
+    method = cupyx.scipy.signal.choose_conv_method(a, v, mode)
```

Comment:
Could you write a comment why we use `v` instead of `v.conj()[::-1]` here?
https://github.com/cupy/cupy/pull/3525#discussion_r448350880